cypress/integration/selectedMovieView_spec.js

```diff
@@ -222,10 +222,34 @@ describe("Selected Movie View", () => {
         cy.get("button")
             .contains("Return Home")
             .click()
-            .url().should("include", "/1")
     })
 
-    //test the functionality of the go back button
+    it("should be able to click on main title to return home ", () => {
+        cy.intercept("GET", "https://rancid-tomatillos.herokuapp.com/api/v2/movies/1", {
+            statusCode: 201,
+            body: { movie: {
+                    "id": 1,  
+                    "title": "Fake Movie Title", 
+                    "poster_path": "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg", 
+                    "backdrop_path": "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg", 
+                    "release_date": "2019-12-04", "overview": "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!", 
+                    "average_rating": 6, 
+                    "genres": ["Drama"], 
+                    "budget": 63000000, 
+                    "revenue": "100,853,753",
+                    "runtime": 139, 
+                    "tagline": "It's a movie!" 
+                }}
+        });
+    
+        cy.visit('http://localhost:3000')
+            .get("#1")
+            .click()
+        cy.get("button")
+            .contains("Rancid Tomatillos")
+            .click({force: true})
```

Comment:
Does this force: true convince Cypress that this is a clickable entity? Very clever way to get Cypress to test what we need it to!